src/renderer/components/reader/ReaderOptions.tsx

```diff
@@ -100,58 +98,57 @@ export class ReaderOptions extends React.Component<Props> {
     }
 
     private themeContent() {
-        const {__} = this.props;
-
+        const {__, settings} = this.props;
+        const withoutTheme = !settings.sepia && !settings.night;
         return (
-            <>
-                <div ref={this.sectionRefList[0]} id={styles.themes_list}>
-                    <div>
-                        <input
+            <div id={styles.themes_list}>
+                <div>
+                    <input
                         id={"radio-" + themeType.Without}
                         type="radio"
                         name="theme"
                         onChange={() => this.handleChooseTheme(themeType.Without)}
-                        {...(!this.props.settings.sepia && !this.props.settings.night
-                        && {checked: true})}
-                        />
-                        <label htmlFor={"radio-" + themeType.Without}>
-                            {this.props.settings.predefined && <SVG svg={DoneIcon} ariaHidden/>}
```

Comment:
I think the line of code `values.predefined = neutral;` must be removed too, in the `handleChooseTheme()` function. Is that correct?

Reply:
Any thoughts about `predefined`? Isn't this completely unused now? (I could not find any references using text search in the Thorium codebase)

Reply:
I already removed it

Reply:
My apologies. Because of the Git "forced pushes" I missed the commit. Thank you! :)